bindings/nodejs/index.d.ts

```diff
@@ -3,17 +3,55 @@
 
 /* auto-generated by NAPI-RS */
 
-export class OperatorFactory {
-  static memory(): Operator
+export const enum ObjectMode {
+  /** FILE means the object has data to read. */
+  FILE = 0,
+  /** DIR means the object can be listed. */
+  DIR = 1,
+  /** Unknown means we don't know what we can do on this object. */
+  Unknown = 2
+}
+export class Memory {
+  constructor()
+  build(): Operator
+}
+export class Operator {
+  object(path: string): DataObject
 }
 export class ObjectMeta {
   location: string
   lastModified: number
   size: number
 }
-export class Operator {
-  meta(path: string): Promise<ObjectMeta>
-  write(path: string, content: Array<number>): Promise<void>
-  read(path: string): Promise<Array<number>>
-  delete(path: string): Promise<void>
+export class ObjectMetadata {
+  /** Mode of this object. */
+  mode: ObjectMode
+  /** Content-Disposition of this object */
+  contentDisposition?: string
+  /** Content Length of this object */
+  contentLength?: number
+  /** Content MD5 of this object. */
+  contentMd5?: string
+  /** Content Range of this object. */
+  contentRange?: Array<number>
+  /** Content Type of this object. */
+  contentType?: string
+  /** ETag of this object. */
+  etag?: string
+  /** Last Modified of this object. */
+  lastModified: number
+}
+export class ObjectLister {
+  nextPage(): Promise<Array<DataObject>>
+}
+export class DataObject {
```

Comment:
Oh, I finally get it. `Object` in nodejs is a keyword...

We should think about https://github.com/datafuselabs/opendal/issues/1475

As for now, let's keep this.

Reply:
No. This may be a `NAPI-RS`'s bug. I'm preparing to dig out.

Reply:
I don't know if we can declare a type called `Object`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object

Reply:
Oh, at least it's ok in a browser.

```js
class Object {
  constructor(height, width) {
    this.area = height * width;
  }
}

new Object(1,2)
```

Reply:
However, it's better not to do so. We can choose another name instead.

Reply:
For now, I'm planning to keep this. The name of this type doesn't have a particularly significant impact on users' perceptions.